akka-docs-dev/rst/scala/http/client-side/host-level.rst

```diff
@@ -0,0 +1,147 @@
+.. _HostLevelApi:
+
+Host-Level Client-Side API
+==========================
+
+As opposed to the :ref:`ConnectionLevelApi` the host-level API relieves you from manually managing individual HTTP
+connections. It autonomously manages a configurable pool of connections to *one particular target endpoint* (i.e.
+host/port combination).
+
+
+Requesting a Host Connection Pool
+---------------------------------
+
+The best way to get a hold of a connection pool to a given target endpoint is the ``Http.cachedHostConnectionPool(...)``
+method, which returns a ``Flow`` that can be "baked" into an application-level stream setup. This flow is also called
```

Comment:
this explanation is a bit different from what was used in the connection-level doc section. I think it can be useful to sync up these sections so they look similar in the introduction parts.